### PR TITLE
titan_wgs, titan_wgs_utils seperated

### PR DESCRIPTION
**Pull request by DouglasAbrams**

### CODE REVIEW

File: titan_wgs_utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - 'r-domc'
+    - 'r >=3.1.0'
+    - python
+    - perl >=5.22.0
+    - numpy
+    - pandas
+test:
+  commands:
+    - '$R -e "library(''TitanCNA'')"'
+    - 'createTITANsegmentfiles.pl -h | cat'
```
Comment (diljotgrewal) [inline titan_wgs_utils/meta.yaml]: this'll break??

File: titan_wgs_utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - 'parse_titan_segments.py -h'
```
Comment (diljotgrewal) [inline titan_wgs_utils/meta.yaml]: this too??

File: titan_wgs_utils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+about:
+  home: http://bioconductor.org/packages/HMMcopy/
```
Comment (diljotgrewal) [inline titan_wgs_utils/meta.yaml]: titan here please?

Comment (DouglasAbrams) [reply titan_wgs_utils/meta.yaml]: shoulld it be changed in titan_wgs as well?

**Pull request merged**